qiita_pet/util.py

```diff
@@ -50,3 +50,21 @@ def clean_str(item):
         cleaned string
     """
     return str(item).replace(" ", "_").replace(":", "")
+
+
+def generate_param_str(param):
+    """Generate an html string with the parameter values
+
+    Parameters
+    ----------
+    param : BaseParameters
+        The parameter to generate the str
+
+    Returns
+    -------
+    str
+        The html string with the parameter set values
+    """
+    result = ["<b>%s:</b> %s" % (name, value)
```

Comment:
IMHO using a table would make more sense in this case, not sure that this should block this PR though.